import/csv-importer.php

```diff
@@ -0,0 +1,94 @@
+    $keys = array_slice(explode(";", trim($tableHeaderItems[0])), 0, 8); //eight table header items
+
+    foreach ($fileContent as $item) {
+        $itemArray = explode(";", trim($item));
+        $itemArrayWithKeys = array_combine($keys, $itemArray);
+
+        echo "\"" . $itemArrayWithKeys["id"] . "\" => new JobCardDefinition(\n";
+        echo "\t" . "id: new CardId('" . $itemArrayWithKeys["id"] . "'),\n";
+        echo "\t" . "title: '" . $itemArrayWithKeys["title"] . "',\n";
+        echo "\t" . "description: '" . $itemArrayWithKeys["description"] . "',\n";
+        echo "\t" . "phaseId: LebenszielPhaseId::PHASE_" . $itemArrayWithKeys["phase"] . ",\n";
+        echo "\t" . "year: new Year(" . $itemArrayWithKeys["year"] . "),\n";
+        echo "\t" . "gehalt: new MoneyAmount(+" . $itemArrayWithKeys["gehalt"] . "),\n";
+        echo "\t" . "requirements: new JobRequirements(\n";
+        echo "\t\t" . "zeitsteine: 1,\n";
+        echo "\t\t" . "bildungKompetenzsteine: " . $itemArrayWithKeys["minBildungUndKarriere"] . ",\n";
+        echo "\t\t" . "freizeitKompetenzsteine: " . $itemArrayWithKeys["minSozialesUndFreizeit"] . ",\n";
+        echo "\t),\n),\n";
+    }
+}
+
+function importWeiterbildungCards() {
+    echo "\n\n--WEITERBILDUNG CARDS--\n\n";
+    $file = file(__DIR__ . "/Weiterbildung-Table 1.csv");
+    $fileContent = array_slice($file, 2); //removes the first two elements (table name and table header)
+    $tableHeaderItems = array_slice($file, 1, 1); //array element containing the table headers
+    $keys = array_slice(explode(";", trim($tableHeaderItems[0])), 0, 3); // first three table header items
+    $answerIds = ["a", "b", "c", "d"];
+
+    foreach ($fileContent as $item) {
+        //first answer Id is used for the correct answer -> randomized through the shuffle
+        shuffle($answerIds);
+        $itemArray = explode(";", trim($item));
+        $itemArrayUpdated = array_slice($itemArray, 0, 3); //remove all wrong answers
+        $itemArrayWithKeys = array_combine($keys, $itemArrayUpdated);
+
+        $wrongAnswersArray = array_filter(array_slice($itemArray, 3));
+        $itemArrayWithKeys["wrongAnswers"] = $wrongAnswersArray; //add wrong answers stored in an array
+
+        echo "\"" . $itemArrayWithKeys["id"] . "\" => new WeiterbildungCardDefinition(\n";
+        echo "\t" . "id: new CardId('" . $itemArrayWithKeys["id"] . "'),\n";
+        echo "\t" . "description: '" . $itemArrayWithKeys["description"] . "',\n";
+        echo "\t" . "answerOptions: [\n";
+        echo "\t\t" . "new AnswerOption(new AnswerId(\"" . $answerIds[0] . "\"), \"" . $itemArrayWithKeys["correctAnswer"] . "\", true),\n";
+
+        foreach($itemArrayWithKeys["wrongAnswers"] as $wrongAnswer) {
+            $key = array_search($wrongAnswer, $itemArrayWithKeys["wrongAnswers"]);
+            echo "\t\t" . "new AnswerOption(new AnswerId(\"" . $answerIds[$key + 1] . "\"), \"" . $wrongAnswer . "\"),\n";
```

Comment:
Du kannst dir den Key auch einfacher holen
```suggestion
        foreach($itemArrayWithKeys["wrongAnswers"] as $key => $wrongAnswer) {
```